src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -49,31 +49,31 @@ class SuggestionProvider extends React.Component {
         'files',
         'io.cozy.files'
       )
-      const files = allDocs.data
 
+      const files = allDocs.data
       const folders = files.filter(file => file.type === TYPE_DIRECTORY)
 
-      const normalizedFiles = files
-        .filter(file => file.trashed !== true)
-        .map(file => {
-          const isDir = file.type === TYPE_DIRECTORY
-          const dirId = isDir ? file._id : file.dir_id
-          let path
-          if (isDir) {
-            path = file.path
-          } else {
-            const parentDir = folders.find(folder => folder._id === file.dir_id)
-            path = parentDir && parentDir.path ? parentDir.path : ''
-          }
-
-          return {
-            id: file._id,
-            name: file.name,
-            path,
-            url: window.location.origin + '/#/folder/' + dirId,
-            icon: getIconUrl(file)
-          }
-        })
+      const notInTrash = file => file.trashed !== true
```

Comment:
Why did not you use a more classical approach as explain in #730 comment:

> I build a jsbin to illustrate what could be used to return true if value is `undefined`: https://jsbin.com/wurodamupa/edit?js,console

```js
const arr = [ true, false, undefined, null]

console.log(arr.map(val => val))
// [true, false, undefined, null]
console.log(arr.map(val => !val))
// [false, true, true, true]
console.log(arr.map(val => !!val))
// [true, false, false, false]
console.log(arr.map(val => val === false))
// [false, true, false, false]
console.log(arr.map(val => val !== true))
// [false, true, true, true]
```

To be more clear `val => !val` is returning the exact same result as the non straigth forward `val => val !== true` event on `null` and `undefined`.

But I may miss something?

Reply:
You're not, I read it too quickly, my bad. There's another change I have to make, I'll incorporate that change in it.